codespell_lib/data/dictionary.txt

```diff
@@ -7216,6 +7264,8 @@ recurrance->recurrence
 recursivion->recursion
 recursivly->recursively
 recusion->recursion
+recusive->recursive
```

Comment:
or "reclusive"

Reply:
nice. Added + applied suggestion to other similar words